rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -249,6 +249,73 @@ def testQuery_noSummaries(self):
     response = self.send_get('ParticipantSummary')
     self.assertBundle([], response)
 
+  def test_get_summary_list_returns_total(self):
+    page_size = 10
+    num_participants = 20
+    SqlTestBase.setup_codes([PMI_SKIP_CODE], code_type=CodeType.ANSWER)
+    questionnaire_id = self.create_demographics_questionnaire()
+
+    # Prove that no results means a total of zero (if requested)
+    response = self.send_get('ParticipantSummary?_count=10&_includeTotal=true')
```

Comment:
Use `page_size` here (and in place of the other hard-coded 10's below)?

Reply:
Done